core/validation/pre_transformation_validate.py

```diff
@@ -0,0 +1,168 @@
+
+
+def authorisation_validation(workbook: dict[str, pd.DataFrame], place_names: tuple, fund_types: tuple, schema: dict):
+    """Performs authorisation validation based on the provided schema.
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas
+                    DataFrames representing each sheet in the submission.
+    :param place_names: A tuple of valid place names.
+    :param fund_types: A tuple of valid fund types.
+    :param schema: The schema containing authorisation checks.
+    :raises ValidationError: If authorisation validation fails.
+    """
+    type_to_authorised_values = {
+        "Place Name": place_names,
+        "Fund Type": fund_types,
+    }
+    authorisation_checks = list()
+    for sheet, column, row, expected_values, type in schema["authorisation_checks"]:
+        expected_values = type_to_authorised_values[type]
+
+        authorisation_checks.append(
+            PreTransformationCheck(sheet=sheet, column=column, row=row, expected_values=expected_values, type=type)
+        )
+
+    failures = check_values(workbook, authorisation_checks, "unauthorised_submission_failure")
+
+    if failures:
+        raise ValidationError(validation_failures=failures)
+
+
+def wrong_input_validation(workbook: dict[str, pd.DataFrame], schema: dict):
+    """Performs wrong input validation based on the provided schema.
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas
+                     DataFrames representing each sheet in the submission.
+    :param schema: The schema containing wrong input checks.
+    :raises ValidationError: If wrong input validation fails.
+    """
+    wrong_input_checks = schema["wrong_input_checks"]
+
+    failures = check_values(workbook, wrong_input_checks, "wrong_input_failure")
+
+    if failures:
+        raise ValidationError(validation_failures=failures)
+
+
+def conflicting_input_validation(workbook: dict[str, pd.DataFrame], schema: dict):
+    """Performs conflicting input validation based on the provided schema.
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas
+                     DataFrames representing each sheet in the submission.
+    :param schema: The schema containing conflicting input checks.
+    :raises ValidationError: If conflicting input validation fails.
+    """
+    fund_type_acronym_to_full_name = {"TD": "Town_Deal", "HS": "Future_High_Street_Fund"}
+    conflicting_input_checks = list()
+    for sheet, column, row, expected_values, type in schema["conflicting_input_checks"]:
+        # TODO: if extra conflicting checks introduced, make this this logic dependent on 'type'
+        col, row = expected_values
+        fund_type_of_entered_place = PLACE_TO_FUND_TYPE.get(str(workbook[sheet].iloc[col][row]).strip())
+        expected_values = tuple(map(fund_type_acronym_to_full_name.get, fund_type_of_entered_place))
```

Comment:
I think it's generally clearer to use a comprehension instead of using map 
```
expected_values = tuple(fund_type_acronym_to_full_name.get(fund_type) for fund_type in fund_type_of_entered_place)
```
but I don't really like this because it's so TF specific